core/src/main/java/org/apache/calcite/util/RelToSqlConverterUtil.java

```diff
@@ -196,4 +197,12 @@ public static SqlSpecialOperator specialOperatorByName(String opName) {
       }
     };
   }
+
+  public static void writeAlwaysCondition(SqlWriter writer, boolean value) {
```

Comment:
This method requires some annotations to clarify that some databases require converting True to '1=1' and false to '1=0'.